Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Fleet/API/Onboarding.yaml

```diff
@@ -51,10 +51,10 @@ apis:
             params:
               fleetOwnerId: Text
             query:
-              makeSelfieAadhaarPanMandatory: Bool
-              providePrefillDetails: Bool
-              onboardingVehicleCategory: VehicleCategory
-              driverId: Id Driver
+              - driverId: Id Driver
+              - makeSelfieAadhaarPanMandatory: Bool
+              - onboardingVehicleCategory: VehicleCategory
+              - providePrefillDetails: Bool
```

Comment:
nothing changed in generation, but this syntax is more safe
https://github.com/nammayatri/namma-dsl/blob/main/README.md?plain=1#L122